AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -3551,12 +3565,16 @@ - (NSString *)_recursiveDescriptionHelperWithIndent:(NSString *)indent
 
 - (NSString *)asciiArtString
 {
-    return [ASLayoutSpec asciiArtStringForChildren:@[] parentName:[self asciiArtName]];
+  return [ASLayoutSpec asciiArtStringForChildren:@[] parentName:[self asciiArtName]];
 }
 
 - (NSString *)asciiArtName
 {
-    return NSStringFromClass([self class]);
+  NSString *string = NSStringFromClass([self class]);
+  if (_debugName) {
+    string = [string stringByAppendingString:[NSString stringWithFormat:@" (debugName = %@)",_debugName]];
```

Comment:
Suggest maybe put in "\"%@\"" to have only the _debugName shown directly in quotes - to shorten the asciiArt constructs that are created